blocks/init/src/Blocks/assets/scripts/application-blocks-editor.js

```diff
@@ -23,24 +23,44 @@ import { Wrapper } from '../../wrapper/wrapper';
 import WrapperManifest from '../../wrapper/manifest.json';
 import globalSettings from '../../manifest.json';
 
+const safelyRequireContext = (basePath, searchSubdirectories, regularExpression) => {
```

Comment:
The idea was not to change anything in the existing project's implementation. Can this be done without changing the existing files?

Reply:
Could you please clarify what you mean by "not changing anything in the existing project's implementation"? I have no any other idea  :)

Reply:
We have this file:
https://github.com/infinum/eightshift-frontend-libs/blob/develop/blocks/init/src/Blocks/assets/scripts/application-blocks-editor.js

I want it to remain exactly as is. So all changes you can make is inside the `registerBlocks` and `registerVariations` methods